src/ocaml/utils/misc.ml

```diff
@@ -111,9 +111,19 @@ let remove_file filename =
   with Sys_error _msg -> ()
 
 let rec split_path path acc =
-  match Filename.dirname path, Filename.basename path with
-  | dir, _ when dir = path -> dir :: acc
-  | dir, base -> split_path dir (base :: acc)
+  match Filename.dirname path with
+  | dir when dir = path ->
+    let is_letter c = (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') in
+    let dir =
+      if String.length dir > 2 && is_letter dir.[0] && dir.[1] = ':'
+      then
+        Printf.sprintf "%s%s"
+          (String.sub dir ~pos:0 ~len:2 |> String.uppercase_ascii)
+          Filename.dir_sep
+      else dir
```

Comment:
This only normalizes the backslash following the  drive letter. Are we sure this is enough? I would have thought that under Windows we should normalize all slashes. I will try it and report back.

Reply:
Other slashes were already normalized as expected: `split_path` returns a list of path segments which are concatenated back using `Filename.concat`.